packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/quanticGeneratedAnswer.js

```diff
@@ -297,16 +315,16 @@ export default class QuanticGeneratedAnswer extends LightningElement {
    */
   async handleDislike(event) {
     event.stopPropagation();
-    this.generatedAnswer.dislike?.();
-    if (!this.feedbackSubmitted) {
+    if (!this._disliked) {
+      this._disliked = true;
+      this._liked = false;
+      this.generatedAnswer.dislike?.();
       // @ts-ignore
-      await FeedbackModal.open({
-        label: this.labels.feedback,
+      await FeedbackModalQna.open({
```

Comment:
if we are no longer opening the feedbackModal, but the FeedbackModalQna here shouldn't we remove the options getter and associated labels?

Reply:
wut?